<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminMemberEdit</class>
 <widget class="QMainWindow" name="adminMemberEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: #FFE5D0;
	font-family: Montserrat;
}</string>
   </property>
   <widget class="QLabel" name="logoImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/images/bblLogo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-weight: bold;
	font-size: 14px;

}</string>
    </property>
    <property name="text">
     <string>Edit Member Details</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>191</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="memberIcon">
       <property name="text">
        <string>back</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="memberBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>911</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border-radius: 25px;
	background: rgb(255, 255, 255);
}

/*QLineEdit{
	background:rgb(255, 229, 208);
	border-radius: 15px;
}*/

QLabel{
	background: rgb(255, 255, 255);
	font-size: 14px;
	font-weight: bold;

}

QLineEdit {
	background:rgb(255, 229, 208);
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
}

QDateEdit{
	background:rgb(255, 229, 208);
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
}

QFormLayout{
	color: rgb(255, 255, 255);
}


QPushButton {
	border-radius: 15px; 
	font-size: 13px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="updateMemberButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: #E78A6B;
color:  #fff;
font-weight: 500;
}

QPushButton:hover {
background-color: rgba(231, 138, 107, 0.8);
}

QPushButton:pressed {
background-color: rgb(196, 116, 90);
}</string>
     </property>
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="mobileNumLineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>127</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>238</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	padding: 5px;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>164</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777185</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Date of Birth (DD/MM/YYYY)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>127</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile Number</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>164</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>201</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>201</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777185</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>235</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777185</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Full Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="firstNameLineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>51</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>21</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line{
	color:rgb(221, 46, 68);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
